dallinger/command_line/docker_ssh.py

```diff
@@ -327,15 +327,16 @@ def wrapper(*args, **kwargs):  # pragma: no cover
                 tmp_dir, config.get("docker_image_base_name"), out=Output()
             )
 
-            if not local_build and not push_build:
+            remote_build = not local_build
+            if remote_build and not push_build:
                 # If built remotely and not pushing, the image is only on the remote daemon.
                 # We need to get its full name (repo:tag) for deployment.
                 # The build_image function already returns the image name, so we use that.
                 print(
                     f"Image {image_name} built remotely, skipping push to registry because --push-build was not selected."
                 )
             else:
-                # If not remote_build, or remote_build and push_remote_build, then push.
+                # If it's a local build, or if it's a remote build and and push_remote_build, then push.
```

Comment:
This should be `push_build` instead of `push_remote_build` I guess

Reply:
Otherwise the `psynet deploy ssh` now succeeded using this branch

Reply:
Oh well spotted! Great